src/opengl.cpp

```diff
@@ -1935,7 +1940,7 @@ void glDrawWorld(view_t* camera, int mode)
 		glEnable(GL_BLEND);
 
 		// first (higher) sky layer
-		glColor4f(1.f, 1.f, 1.f, .5);
+		glColor4f(1.f, 1.f, 1.f, getLightAtModifier);
```

Comment:
Fine to remove the .5?

Reply:
yeah i just decided i didn't like the top layer of sky being transparent ... the two layers combined weren't opaque